akka-actor-typed/src/main/scala/akka/actor/typed/internal/entity/EntityEnvelopeSerializer.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed.internal.entity
+
+import akka.actor.typed.EntityEnvelope
+import akka.actor.typed.internal.protobuf.EntityMessages
+import akka.annotation.InternalApi
+import akka.event.Logging
+import akka.protobufv3.internal.UnsafeByteOperations
+import akka.serialization.BaseSerializer
+import akka.serialization.DisabledJavaSerializer
+import akka.serialization.SerializationExtension
+import akka.serialization.SerializerWithStringManifest
+import akka.serialization.Serializers
+import akka.util.ByteString
+import akka.util.ByteString.ByteString1
+import akka.util.ByteString.ByteString1C
+import akka.protobufv3.internal.{ ByteString => ProtoByteString }
+
+/**
+ * INTERNAL API
+ *
+ * Serializer for [[EntityEnvelope]].
+ */
+@InternalApi private[akka] final class EntityEnvelopeSerializer(val system: akka.actor.ExtendedActorSystem)
```

Comment:
There is a bunch of code copied from remote here. I think I will be able to remove all of this.